### ISSUE TITLE
ad creative Url Tracking Params Update

### ISSUE THREAD

**Issue by ricky-cortopassi** [facebook/facebook-php-business-sdk#600]
https://developers.facebook.com/docs/marketing-api/reference/ad-creative#Updating

-- not so much an issue but quality of life improvement. 
It would be nice if were able to update an ad creative url tracking params, instead of needing to copy the prev make cahanges and create new one to apply to ad.. 
 The only fields that seem to be able to update on the creative are the adlabels, name ,status. 

-- 1 reason behind needed to update these sometimes is that a change in something like the campaign name, ad set name... etc.. 
is not reflected in the dynamic params {{campaign.name}} , {{adset.name}} .. 
These will always reference the ORGINAL names when created, so to keep this correct , sometimes need to update these to the raw string values of the changed names so that they are reflected as the current state when served in query string params for utm_campaign .. and other needed params .

**Comment by tumispro** [comment]
Agreed! Also: I am stuck changing the creative of an existing ad. $ad->createSelf() seems to support the creative field while $ad->updateSelf() doesn't. Does anyone here by any chance know how to update the linked creative?
